sdk/anomalydetector/ai-anomaly-detector-rest/src/paginateHelper.ts

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PagedAsyncIterableIterator, PagedResult, getPagedAsyncIterator } from "@azure/core-paging";
```

Comment:
@MaryGao Is this where you did exploration on RLCs needing a custom pagination helper?